domainhunter.py

```diff
@@ -595,6 +607,7 @@ def getIndex(cells, index):
         # Print number of domains to perform reputation checks against
         if check:
             print("\n[*] Performing reputation checks for {} domains".format(len(domain_list_unique)))
+	    print("")
```

Comment:
Not Python3 syntax (change to use spaces vs tabs)
```python3 ./domainhunter.py
  File "./domainhunter.py", line 610
    print("")```

Reply:
tabs fixed